assets/prototype/domain/eventEditor/containers/useConfigProvider.ts

```diff
@@ -1,30 +1,34 @@
 import { useContext } from 'react';
 import { pick } from 'ramda';
 import { ConfigContext } from '../../../infrastructure/services/contextProviders/ConfigProvider';
+import getDateTimeFormat from '../../../application/value-objects/date-time/getDateTimeFormat';
 
 type useConfigProviderProps = {
-	dateFormat: boolean;
-	email: boolean;
-	timezone: boolean;
+	dateTimeFormat: boolean;
 };
 
 /**
  *
  * useConfigProvider is used to cherry-pick the exact configs we might need from ConfigProvider.
  */
 
-const useConfigProvider = (props: useConfigProviderProps) => {
-	const config = useContext(ConfigContext);
+const useConfigProvider = ({ dateTimeFormat }: useConfigProviderProps) => {
+	const { dateFormat, timeFormat } = useContext(ConfigContext);
+	let config: any = {};
 
-	const configArray = Object.entries(props).reduce((configs, [configName, configBool]) => {
-		if (configBool) {
-			configs.push(configName);
-		}
+	if (dateTimeFormat) {
+		config.dateTimeFormat = getDateTimeFormat({ dateFormat, timeFormat });
```

Comment:
does this need to be updated as well to account for the combined `dateTimeFormat` being added?

Reply:
yes, my bad here, didn't fully finish all